src/parser.rs

```diff
@@ -593,6 +593,13 @@ impl Library {
         let name = try!(attrs.get("name").ok_or_else(|| mk_error!("Missing function name", parser)));
         let c_identifier = attrs.get("identifier").or_else(|| attrs.get("type"));
         let kind = try!(FunctionKind::from_str(kind_str).map_err(|why| mk_error!(why, parser)));
+        let version = match attrs.get("version") {
+            Some(v) => {
+                println!("Version: {}", v);
+                Some(try!(v.parse().map_err(|why| mk_error!(why, parser))))
+                },
```

Comment:
The Rust style asks us to align the closing brace one tab to the left (probably a typo?) and, surprisingly, not add the trailing comma.